indico/web/templates/forms/_form.html

```diff
@@ -9,13 +9,14 @@
 {%- endmacro %}
 
 {% macro form_header(form, method='post', action='', orientation='horizontal', id=none, classes='', multipart=false,
-                     i_form=true, no_ajax=false) -%}
+                     i_form=true, no_ajax=false, ajax_form=false) -%}
```

Comment:
Well, we do have the caller in there so we could add all additional attributes. But then half of the ajax form related attributes will be put through the form_header parameters and others through the caller. I don't have a strong opinion.